CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Restrict resource spans to script and css only. ([#2623](https://github.com/getsentry/relay/pull/2623))
 - Postpone metrics aggregation until we received the project state. ([#2588](https://github.com/getsentry/relay/pull/2588))
 - Scrub random strings in resource span descriptions. ([#2614](https://github.com/getsentry/relay/pull/2614))
+- Collect `http.decoded_response_content_length`. ([#2638](https://github.com/getsentry/relay/pull/2638))
```

Comment:
Please, merge `master` in and update the CHANGELOG moving this record in `Unreleased` section again.

Reply:
@olksdr made the change!